h2o-core/src/main/java/water/fvec/RawChunk.java

```diff
@@ -0,0 +1,21 @@
+package water.fvec;
+
+import water.fvec.CX0Chunk;
+import water.fvec.Chunk;
+import water.fvec.NewChunk;
+import water.udf.TypedChunk;
+
+/**
+ * Representative of TypedChunk in the world of typeless chunks. 
+ * We don't store any data there. We just pretend to be a "regular" chunk,
+ * to satisfy the obsolete API.
+ */
+public class RawChunk extends CX0Chunk {
```

Comment:
I can add more comments, but I believe what is said above explains the purpose.
CX0Chunk contains a bunch of useful functionality.